docker/influxdb/setup-influxdb.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+while ! /usr/bin/pgrep influx >/dev/null; do
+    echo "influx not yet ready"
```

Comment:
Is there a more reliable way to determine this than just the process existing?

Reply:
I was meaning executing  `influx -execute "CREATE DATABASE $INFLUXDB_IML_DB"` directly until it succeed

Reply:
Updated to attempt creating the database until successful.